cmd/registry/cmd/root.go

```diff
@@ -66,6 +58,5 @@ func Command(ctx context.Context) *cobra.Command {
 	cmd.AddCommand(upload.Command(ctx))
 	cmd.AddCommand(vocabulary.Command(ctx))
 
-	cmd.PersistentFlags().StringVar(&logID, "log-id", "", "Assign an ID which gets attached to the log produced")
```

Comment:
Was this option fully removed? If I run the registry tool from the command line, is there a way for me to know which ID in the server logs corresponds to the run that I just made?

Reply:
Yeah it's fully removed. When we run a command there should always be at least one log produced by the command that can identify the ID for users. If there are any exceptions I think we should add some logging to those so users know something is happening.

In general I think it would be helpful to add at least one general-purpose log "hook" like the inceptor we use in the server. It could simply be a post-run "Success" or "Fail" with some metrics like duration attached, similar to the server.